lib/govuk_app_config/govuk_content_security_policy.rb

```diff
@@ -91,6 +90,16 @@ def self.build_policy(policy)
   def self.configure
     Rails.application.config.content_security_policy_report_only = ENV.include?("GOVUK_CSP_REPORT_ONLY")
 
+    # Sets a nonce per request that is set in the CSP directives for script_src and
+    # style_src.
```

Comment:
Is it worth adding a note like "(but it will have no effect on the latter until we remove unsafe-inline support for it)"?